2023/incoming/issue-290.pod

```diff
@@ -0,0 +1,404 @@
+ForgeBike InitialPaint Peddles Gears RearWheel Chain HandleBars RearBrakes FrontWheel Seat FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint Peddles Gears RearWheel Chain HandleBars RearBrakes FrontWheel FrontBrakes Seat Buddy2TestBike
+ForgeBike InitialPaint Peddles Gears RearWheel Chain HandleBars RearBrakes Seat FrontWheel FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint Peddles Gears RearWheel Chain HandleBars FrontWheel RearBrakes Seat FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint Peddles Gears RearWheel Chain HandleBars FrontWheel RearBrakes FrontBrakes Seat Buddy2TestBike
+...
+ForgeBike InitialPaint HandleBars Gears FrontWheel RearWheel Peddles RearBrakes Chain Seat FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel RearWheel Peddles RearBrakes Chain FrontBrakes Seat Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel RearWheel Peddles Chain RearBrakes Seat FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel RearWheel Peddles Chain RearBrakes FrontBrakes Seat Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel Peddles RearWheel RearBrakes Chain Seat FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel Peddles RearWheel RearBrakes Chain FrontBrakes Seat Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel Peddles RearWheel Chain RearBrakes Seat FrontBrakes Buddy2TestBike
+ForgeBike InitialPaint HandleBars Gears FrontWheel Peddles RearWheel Chain RearBrakes FrontBrakes Seat Buddy2TestBike
+
+=end text
+
+It would still be useful, one may suppose, if Lucas could only enumerate
+the steps in a properly ordered way. But alas, C<Sub::Genius> is far more
+useful than that!
+
+It so happens that if the C<sub>s with the same names as the I<steps> of the
+plan are defined within the scope, then the plan may be actually executed. This
+allows for each step in the plan to have actions associated with them (even
+generating additional nested plans).
+
+Because creating scripts to be executed according to the plan is a tedious
+task, one of the useful tools C<Sub::Genius> provides is called C<stubby>.
+This utility can be used to I<stub> another Perl script with the C<sub> already
+in place, just waiting to be filled with useful purpose.
+
+E.g., if the I<plan> above is saved to a file, let's call it C<./bike.plan>,
+we're able to generate a Perl script that contains subroutine stubs for each of
+the steps that appear in the plan at least once.
+
+  shell> stubby init -f ./bikes.plan --run once > bikes.pl
+
+The file C<bikes.pl> is now filled with subroutines stubs that are waiting to
+be filled in with meaningful code, the script will look something like the following:
+
+=begin perl
+
+use strict;
+use warnings;
+use Sub::Genius;
+
+my $plan = <<EOP;
+  ForgeBike
+  InitialPaint
+  (
+    ( ( RearWheel & Peddles & Gears ) ( RearBrakes & Chain))
```

Comment:
I think we'll need to `s/Peddles/Pedals/` on the whole document.

Reply:
I thought it looked funny. lol